web/src/global.d.ts

```diff
@@ -27,6 +27,7 @@ interface JQuery {
     get_offset_to_window(): DOMRect;
     tab(action?: string): this; // From web/third/bootstrap
     modal(action?: string): this; // From web/third/bootstrap
+    carousel(options: unknown): void; // From web/third/bootstrap
```

Comment:
Let's move this commit to a separate PR; we're deleteing the carousel in https://github.com/zulip/zulip/pull/24301 and I'd prefer to merge that before this commit.

Reply:
Did in a separate commit. Please check.